### PR TITLE
Makes bomb armor lessen the chance of bomb knockout

### PR DESCRIPTION
**Pull request by silicons**
Come on now, fireball's had its reign of terror long enough, and so has IED spears.
:cl:
experimental: Bomb armor is now effective in lessening the chance of being knocked out by bombs.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -401,7 +401,7 @@
 			damage_clothes(200 - bomb_armor, BRUTE, "bomb")
 			if (!istype(ears, /obj/item/clothing/ears/earmuffs))
 				adjustEarDamage(30, 120)
-			if (prob(70))
+			if (prob(min(70 - bomb_armor, 0)))
```
Comment (XDTM) [inline code/modules/mob/living/carbon/human/human_defense.dm]: this is always 0, you should use max

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 2]
```diff
@@ -411,7 +411,7 @@
 			damage_clothes(max(50 - bomb_armor, 0), BRUTE, "bomb")
 			if (!istype(ears, /obj/item/clothing/ears/earmuffs))
 				adjustEarDamage(15,60)
-			if (prob(50))
+			if (prob(min(50 - bomb_armor, 0)))
```
Comment (XDTM) [inline code/modules/mob/living/carbon/human/human_defense.dm]: same here

**Comment by RemieRichards** [comment]
Make only half of the bomb armour count, it already does some of the job.
(so ```- bomb*armour*0.5```)

File: code/modules/mob/living/carbon/human/human_defense.dm (hunk 1 above)
Comment (kevinz000) [inline code/modules/mob/living/carbon/human/human_defense.dm]: FUCK

**Comment by kevinz000** [comment]
K.

**Comment by KorPhaeron** [comment]
he buff heself

**Pull request merged**